test/integration/targets/mysql_db/tasks/encoding_dump_import.yml

```diff
@@ -0,0 +1,67 @@
+---
+
+- set_fact: 
+    latin1_file1="{{tmp_dir}}/{{file}}"
+
+- name: Deleting Latin1 encoded Database
+  mysql_db:
+    name: '{{ db_latin1_name }}'
+    state: absent
+    login_unix_socket: '{{ mysql_socket }}'
+
+- name: create Latin1 encoded database
+  mysql_db:
+    name: '{{ db_latin1_name }}'
+    state: present
+    encoding: latin1
+    login_unix_socket: '{{ mysql_socket }}'
+
+- name: create a table in Latin1 database
+  command: mysql {{ db_latin1_name }} '-e create table testlatin1(id int, name varchar(100));'
+
+- name: inserting data into Latin1 database
```

Comment:
Maybe should you add a small comment warning that the inserted string will be checked later and that changing it requires to update the below test.

Reply:
added